src/tdastro/astro_utils/passbands.py

```diff
@@ -643,3 +880,68 @@ def fluxes_to_bandflux(
         integrand = flux_density_matrix * self.processed_transmission_table[:, 1]
         bandfluxes = scipy.integrate.trapezoid(integrand, x=self.waves, axis=w_axis)
         return bandfluxes
+
+    def plot(self, ax=None, figure=None, color=None, plot_loaded=False):
+        """Plot the passband.
+
+        Parameters
+        ----------
+        ax : matplotlib.pyplot.Axes or None, optional
+            Axes, None by default.
+        figure : matplotlib.pyplot.Figure or None
+            Figure, None by default.
+        color : str or None, optional
+            The color of the curve.
+        plot_loaded : bool
+            Also plot the loaded table as a dashed line. Used for debugging.
+        """
+        if ax is None:
+            if figure is None:
+                figure = plt.figure()
+            ax = figure.add_axes([0, 0, 1, 1])
+
+        # If the color is provided, we use that. Otherwise we try
+        # the LSST filter colors (or default to black).
+        if color is None:
+            color = _lsst_filter_plot_colors.get(self.filter_name, "black")
+
+        ax.plot(
+            self.processed_transmission_table[:, 0],  # X values are the wavelength
+            self.processed_transmission_table[:, 1],  # Y values are the transmission values.
+            color=color,
+            label=self.full_name,
+        )
+        if plot_loaded:
+            ax.plot(
+                self._loaded_table[:, 0],  # X values are the wavelength
+                self._loaded_table[:, 1],  # Y values are the transmission values.
+                color=color,
+                linestyle="--",
+            )
+
+        ax.set_xlabel("Wavelength (A)")
```

Comment:
It works even with no TeX installation:
```suggestion
        ax.set_xlabel(r"Wavelength, $\AA$")
```